packages/@o3r/core/src/utils/deep-fill/deep-fill.spec.ts

```diff
@@ -12,6 +12,20 @@ describe('Deep fill function', () => {
     expect(deepFill(base, source)).toEqual(base);
   });
 
+  it('should support "null" value in source', () => {
+    const base = { selection: { field: 'test-field' }, a: 'string' };
+    const source = { selection: null } as any;
+
+    expect(deepFill(base, source)).toEqual(base);
```

Comment:
it this really what we expect from a deepFill?
it would make sense to me that `deepFill({a: 1}, {a: null})` would result in `{a: null}`
`deepFill({a: 1}, {a: undefined})` results in `{a: 1}`
`deepFill({a: null}, {a: 1})` results in `{a: 1}`